copier.yaml

```diff
@@ -17,11 +17,6 @@ use_apple_silicon:
   help: Do you use Apple Silicon as a GPU?
   default: no
 
-use_windows:
```

Comment:
Please remove the Windows option from the niceML Documentation, mainly in the Getting Started Section.